content/v2.0/reference/api/client-libraries/go.md

```diff
@@ -1,12 +1,204 @@
 ---
-title: InfluxDB Go client library
+title: Go client library
+seotitle: InfluxDB Go client library
 list_title: Go
-description: Use the Go client library to interact with InfluxDB.
-external_url: https://github.com/influxdata/influxdb-client-go
+description: >
+  Use the Go client library to interact with InfluxDB.
 menu:
   v2_0_ref:
     name: Go
     parent: Client libraries
-    url: https://github.com/influxdata/influxdb-client-go
+v2.0/tags: [client libraries, Go]
 weight: 201
 ---
+
+Use the [InfluxDB Go client library](https://github.com/influxdata/influxdb-client-go) to integrate InfluxDB into Go scripts and applications.
+
+This guide presumes some familiarity with Go and InfluxDB.
+If just getting started, see [Get started with InfluxDB](/v2.0/get-started/).
+
+## Before you begin
+   
+1. Install Go 1.3 or later](https://golang.org/doc/install)
+2. Download the client package in your $GOPATH and build the package.
+
+    ```sh
+    # Download the InfluxDB Go client package
+    go get github.com/influxdata/influxdb-client-go
+
+    # Build the package
+    go build
+    ```
+3. Ensure that InfluxDB is running and you can connect to it.
+   If running InfluxDB locally, visit http://localhost:9999.
+   If using InfluxDB Cloud, visit your [InfluxDB Cloud URL](/v2.0/cloud/urls).
+
+## Boilerplate for the InfluxDB Go Client Library  
+
+Use the Go library to write and query data to and from InfluxDB.
+
+In your Go program, import the necessary packages and specify the entry point of your executable program.
+
+```go
+package main
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	influxdb2 "github.com/influxdata/influxdb-client-go"
+)
+```
+
+Next, define variables for your InfluxDB [bucket](/v2.0/organizations/buckets/), [organization](/v2.0/organizations/), and [token](/v2.0/security/tokens/).
+
+```go
+bucket := "<my-bucket>"
+org := "<my-org>"
+token := "<my-token>"
+//variable to store the url of your local or InfluxDB Cloud instance
+url := "<http://localhost:9999>"
+```
+
+To write data, create the the InfluxDB Go Client and pass in our named parameters: `url` and `token`.
+
+```go
+client := influxdb2.NewClient(url, token)
+```
+
+Create a **write client** with the `WriteApiBlocking` method and pass in your other named parameters: `org` and `bucket`. 
```

Comment:
Create a **write client** with the `WriteApiBlocking` method and pass in ~your other named~ **the `org` and `bucket`** parameters. ~: `org` and `bucket`~.